rust/windows-client/src-tauri/Cargo.toml

```diff
@@ -41,7 +44,7 @@ tracing-panic = "0.1.1"
 zip = { version = "0.6.6", features = ["deflate", "time"], default-features = false }
 rand = "0.8.5"
 windows-implement = "0.52.0"
-known-folders = "1.1.0"
+sadness-generator = "0.5.0"
```

Comment:
?? 🤣

Reply:
It's for simulating crashes lol
It doesn't have to go to production but if anyone else needs to check my work, it lets us do segfaults and other stuff that should trigger the crash handler